db/migrate/20161124205003_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :username, null: false, limit: 50, unique: true
+      t.string :password_digest, null: false
+      t.string :email, null: false, unique: true
```

Comment:
We could put it in both. Sunny said that it's typically best practice to have it in both incase the database outlives the models. It would be easy enough to add it to the model.